### DISCUSSION TITLE
新版校園授權軟體網站

### DISCUSSION THREAD

**Discussion by gnsJhenJie** [1101-GS3073-C/Python-Community-GS3073-C#485] [Show and tell]
最近電算中心上線了一個新版的[校園授權軟體網站](https://software.ncu.edu.tw/ ) (https://software.ncu.edu.tw) ，目前是和[舊版網站](https://ca.ncu.edu.tw)並存。這個網站是由我和另一位同學 @pocper1 開發的，歡迎大家逛逛新網站，找找bug或是提供改進的建議~  (如為資安相關問題，請透過email通知 gnsjhenjie@g.ncu.edu.tw)

**Comment by Chenszuhong** [comment]
感覺還不錯，比較淺顯易懂

**Comment by lx3211** [comment]
介面看起來賞心悅目多了哈哈哈哈
原本真的...

**Comment by 0965256727** [comment]
比較好搞懂這個是怎麼運作的。

**Comment by manipulateed** [comment]
這個也太有幫助了吧～之前的介面比較隱晦點排版也不夠現代化的感覺，能有此次更新好感動

**Comment by williamyen0623** [comment]
太強了吧，是大佬...
版面乾淨，內容清晰，而且風格很溫馨、可愛，大加分！

**Comment by number32984** [comment]
高手就在你我的身邊
有神快拜

**Comment by Ben6uo** [comment]
哇賽 這太強了 感謝大哥的分享

**Comment by skyling0299** [comment]
太好了有新東西
就那個navbar好討厭哈哈哈哈，風格完全不一樣

**Comment by HsuChen88** [comment]
前人種樹，後人當免費仔，謝謝開發者。
